### PR TITLE
Some Allay and Warden related mappings. 

### PR DESCRIPTION
**Pull request by HyperPigeon**
- Mapped fields in `WardenEntity ` inner class to `range` and `positionSource`.
- Mapped ImmutableList field in `AllayEntity` to `itemThrownPitches`.
- Mapped public static void method in `GiveItemsToTargetTask` to `throwItemStackToEntity`.

### CODE REVIEW

File: mappings/net/minecraft/entity/passive/AllayEntity.mapping [hunk 1]
```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/unmapped/C_ueeoxhql net/minecraft/entity/passive/AllayEntity
 	FIELD f_fwzlovqf CAN_DUPLICATE Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_gqwygpsf sculkSensorListener Lnet/minecraft/unmapped/C_wpaexsvv;
 	FIELD f_iqgfejuv MEMORY_MODULES Lcom/google/common/collect/ImmutableList;
+	FIELD f_jxnwapwy itemThrownPitches Lcom/google/common/collect/ImmutableList;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/entity/passive/AllayEntity.mapping]:
```suggestion
	FIELD f_jxnwapwy thrownItemPitches Lcom/google/common/collect/ImmutableList;
```
I think this is the more natural ordering

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Review by supersaiyansubtlety — APPROVED** [review]

**Review by eristhea — APPROVED** [review]

File: mappings/net/minecraft/entity/mob/warden/WardenEntity.mapping [hunk 2]
```diff
@@ -67,3 +67,6 @@ CLASS net/minecraft/unmapped/C_fynaplbn net/minecraft/entity/mob/warden/WardenEn
 		ARG 1 origin
 		ARG 2 source
 		ARG 3 range
+	CLASS C_xfllwcxz
+		FIELD f_eudouoeu range I
```
Comment (eristhea) [inline mappings/net/minecraft/entity/mob/warden/WardenEntity.mapping]:
```suggestion
		FIELD f_eudouoeu RANGE I
```

File: mappings/net/minecraft/entity/passive/AllayEntity.mapping [hunk 3]
```diff
@@ -6,6 +6,7 @@ CLASS net/minecraft/unmapped/C_ueeoxhql net/minecraft/entity/passive/AllayEntity
 	FIELD f_fwzlovqf CAN_DUPLICATE Lnet/minecraft/unmapped/C_rinmcaxy;
 	FIELD f_gqwygpsf sculkSensorListener Lnet/minecraft/unmapped/C_wpaexsvv;
 	FIELD f_iqgfejuv MEMORY_MODULES Lcom/google/common/collect/ImmutableList;
+	FIELD f_jxnwapwy thrownItemPitches Lcom/google/common/collect/ImmutableList;
```
Comment (eristhea) [inline mappings/net/minecraft/entity/passive/AllayEntity.mapping]:
```suggestion
	FIELD f_jxnwapwy THROWN_ITEM_PITCHES Lcom/google/common/collect/ImmutableList;
```

**Review by eristhea — CHANGES_REQUESTED** [review]
fixes for checkstyle!

**Review by eristhea — APPROVED** [review]

**Pull request merged**